release/npm/BUILD

```diff
@@ -18,16 +18,25 @@ genrule(
     name = "package",
     srcs = [
         "//:CONTRIBUTORS",
+        ":npm"
     ],
     outs = ["package.json"],
-    cmd = "$(location :npm) $(location //:CONTRIBUTORS) > $@",
+    cmd = "$(location :package_sh) ./$(location :npm) $(location //:CONTRIBUTORS) > $@",
```

Comment:
Why do we need this indirection? What happens if you go back to the direct invocation of `:npm` from the genrule?

Reply:
The indirection is a workaround for a problem I found when upgrading into the latest `rules_go@0.30.0` however I was able to reproduce and verify the problem as early as upgrading into [v0.23.0](https://github.com/bazelbuild/rules_go/releases/tag/v0.23.0)

Below you can find the error output:

```
bazel-watcher tiagocosta$ bazel build --config=release //release/npm:package
INFO: Analyzed target //release/npm:package (1 packages loaded, 36 targets configured).
INFO: Found 1 target...
ERROR: /projects/bazel-watcher/release/npm/BUILD:17:8: Executing genrule //release/npm:package failed: (Exit 2): bash failed: error executing command /bin/bash -c 'source external/bazel_tools/tools/genrule/genrule-setup.sh; bazel-out/host/bin/release/npm/npm_/npm ./CONTRIBUTORS > bazel-out/darwin-fastbuild/bin/release/npm/package.json'

Use --sandbox_debug to see verbose messages from the sandbox
panic: The version string was not overriden. Please rebuild with --stamp

goroutine 1 [running]:
main.main()
	release/npm/generate_package.go:29 +0x3fc
Target //release/npm:package failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 2.707s, Critical Path: 1.57s
INFO: 6 processes: 4 internal, 2 darwin-sandbox.
FAILED: Build did NOT complete successfully
```

I really think it is related with that was opening by you at https://github.com/bazelbuild/rules_go/issues/2224 that was not reproducible somewhere in between `v0.19.4` and `v0.22.4` but is now a problem again. For the sake of what we are doing here I think we can live with the indirection.

Reply:
It's probably transition related, those things are beasts. Okay, let's merge this and get a release out there for all the M1 owners. Thanks for fighting through this!